.appveyor.yml

```diff
@@ -62,18 +62,15 @@ cache:
 - '%LOCALAPPDATA%\pip\Cache'
 
 artifacts:
-- path: pillow\dist\*.egg
+- path: pillow\*.egg
   name: egg
```

Comment:
The egg format is deprecated on PyPI:

* https://peps.python.org/pep-0715/

I know we're not uploading these, but do we need them here?

```suggestion
```

(Non-blocking for merge.)